src/Hl7.Fhir.Support.Poco/Serialization/SerializerSettings.cs

```diff
@@ -29,6 +29,11 @@ public class SerializerSettings
         /// </summary>
         public bool TrimWhiteSpacesInXml { get; set; } = true;
 
+        /// <summary>
+        /// Include mandatory elements to the given set elements
+        /// </summary>
+        public bool IncludeMandatoryToElements { get; set; } = false;
```

Comment:
Nothing technical, but I did not "get" the name when I read it.  I'll try a few alternatives:

* IncludeMandatoryInElementsSummary
* ElementsShouldIncludeMandatory
* IncludeMandatoryWithElements

The comment:  "Include mandatory elements when serializing a subset of chosen elements (_elements). See <see cref="Method with elements overload" />.